core/utils.js

```diff
@@ -37,28 +38,23 @@ const userAgent = goog.require('Blockly.utils.userAgent');
 
 
 /**
- * Don't do anything for this event, just halt propagation.
+ * Halts the propagation of the event without doing anything else.
```

Comment:
Perhaps a bit belt-and-braces, but why not mark this as `@deprecated`?  (Here and everywhere below that calls `deprecation.warn`.)